lightning_pose_app/ui/train_infer.py

```diff
@@ -836,11 +836,11 @@ def _render_streamlit_fn(state: AppState):
             #### Training options
             """
         )
-        expander = st.expander("Change Defaults")
-
+        #expander = st.expander("Change Defaults")
+        expander = st.expander("Expand to adjust the maximum training epochs and the types of unsupervised losses employed for the semi-supervised models")
```

Comment:
Let's reduce this a bit to "Expand to adjust maximum training epochs and select unsupervised losses"